### PR TITLE
author_count: use properly elasticsearch range INSPIR-1468

### PR DESCRIPTION
**Pull request by ammirate**
Related issue: https://its.cern.ch/jira/browse/INSPIR-1468

Signed-off-by: Antonio Cesarano <cesarano2607@gmail.com>

### CODE REVIEW

File: tests/integration/test_search_views.py [hunk 1]
```diff
@@ -379,3 +383,25 @@ def _debug(log_output):
...
+        '$schema': 'http://localhost:5000/schemas/records/hep.json',
+        'document_type': ['article'],
+        'titles': [{'title': 'Article with 10 authors'}],
+        '_collections': ['Literature'],
+        'authors': []
+    }
+    for i in range(10):
+        record_json['authors'].append({'full_name': 'Pincopallino' + str(i)})
+
+    rec = InspireRecord.create(data=record_json)
```
Comment (drjova) [inline tests/integration/test_search_views.py]: I'd use the factory for creating a record :)

Comment (monaawi) [reply tests/integration/test_search_views.py]: I think the factory wouldn't work here if he needs the record to be in ES as well. I am not sure though

Comment (drjova) [reply tests/integration/test_search_views.py]: hmm, It sends the record on ES if you pass ``index_name`` https://github.com/inspirehep/inspire-next/blob/master/tests/integration/helpers/factories/db/invenio_records.py#L120

Comment (monaawi) [reply tests/integration/test_search_views.py]: yes, we looked into that once and it didn't work (when I was working on the tests in test_records_views). And then we decided it was not in the scope of the story and that we will look into it

Comment (ammirate) [reply tests/integration/test_search_views.py]: I confirm, my first attempt used the factory and it did not work.

**Pull request merged**